content/en/error_tracking/rum.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Error Tracking for Web and Mobile Applications
+kind: documentation
+---
+
+{{< whatsnext desc="To get started with Error Tracking for RUM, see the corresponding documentation to upload source maps for your framework:" >}}
```

Comment:
Should this include Roku?

Reply:
Great catch -- the docs I copied from RUM omitted Roku, so I added it to both, thank you!

And yeah, the jumps within RUM are unfortunate but expected, they were already there and I just don't have a great solution for those, especially since it's apparently preferred to reroute mobile folks over to the mobile section of the docs at the first opportunity. So I'm going to leave them, but I'm with you, I find it jarring.